packages/salesforcedx-utils-vscode/src/commands/commandletExecutors.ts

```diff
@@ -177,6 +177,13 @@ export abstract class LibraryCommandletExecutor<T>
           token.onCancellationRequested(() => {
             this.cancelled = true;
             notificationService.showCanceledExecution(this.executionName);
+
+            telemetryService.sendCommandEvent(
```

Comment:
@YousefHaggyHeroku - Can you please explain this change? FYI @randi274

Reply:
Good question, previously we had a `force_function_start_cancelled` metric logging [in the actual `forceFunctionStart code`](https://github.com/forcedotcom/salesforcedx-vscode/blob/158d4794e1a9498c13508b21809848c953fe7588/packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionStart.ts#L107-L111). Since the cancellation request is handled by the library executor now, I figured it might be worth moving that metric here in case other long running Library based-commands exist in the future.

If there's a downside to this, I can move it to `token?.onCancellationRequested` in `forceFunctionStart` instead

Reply:
This is good 👍